moveit_experimental/CMakeLists.txt

```diff
@@ -56,11 +57,10 @@ catkin_package(
     pluginlib
   DEPENDS
     Boost
-    Eigen3
```

Comment:
I was 99% sure moveit_core didn't list Eigen as a depend, but now I see that two months ago @v4hn [added it](https://github.com/ros-planning/moveit/commit/e96898fa02d1f377412b9afd4723b854387821e5#diff-a1e6489257911e9623a4b64c91b728c3R107).

I just looked into it further and the problem is that its not ALL CAPS.

This isn't a huge deal (nothing breaks and this is a experimental package). But I will fix it asap